### PR TITLE
Use Arctis USB Finder NPM Package

### PR DESCRIPTION
**Pull request by richrace**
:wave: So this uses the package I've built which is based on your work to load/find the Arctis USB devices.

I split it out into it's own package as I plan on making an Electron tray version (https://github.com/richrace/arctis-monitor).

The [Arctis USB Finder](https://github.com/richrace/arctis-usb-finder) package also includes support for the Arctis 7X (with some additional fields). 

Let me know what you think. 

Happy for this to be rejected/accepted 😀

### CODE REVIEW

File: percentage.js [hunk 1]
```diff
@@ -1,48 +1,6 @@
...
-  },
-  {
-    name: 'Arctis 9',
-    vendorId: 4152,
-    productId: 0x12c2,
-    reportIdx: 3,
-    writeBytes: [0x06, 0x12],
-  },
-]
+const getHeadphoens = require('arctis-usb-finder').default
```
Comment (atagulalan) [inline percentage.js:3]: nit: typo

**Review by atagulalan — CHANGES_REQUESTED** [review]
Excellent work! There is a small typo, but overall, this is great.

I am currently looking for a maintainer. If you are interested, please reply this comment.

Thank you.

**Review by atagulalan — APPROVED** [review]

File: percentage.js (hunk 1 above)
Comment (richrace) [reply percentage.js]: 🙈  Thanks for fixing this.

**Comment by richrace** [comment]
Thanks for merging 🎉  

I'd happily be a maintainer to bump this dependency when I update it in the future. I just don't have access to a Windows PC (or a VM) at the moment so I won't be extending the functionality or manually testing this.

**Pull request merged**